R/container.R

```diff
@@ -9,8 +9,19 @@
 Container <- R6Class("Container",
   inherit = Entity,
   active = list(
-    containerType = function(value) {
+    containerTypeInt = function(value) {

```

Comment:
@msevestre Does it make sense for you to have two different function for returning int (e.g. for flow-control in user code) or the string (for readability)?

Reply:
OK, implemented the second one.

Reply:
@msevestre Please review the solution.